controllers/pwa.js

```diff
@@ -187,13 +187,28 @@ function renderPwaListPage(req, res) {
   const limit = parseInt(req.query.limit, 10) || LIST_PAGE_SIZE;
   const end = pageNumber * LIST_PAGE_SIZE;
   const contentOnly = false || req.query.contentOnly;
-  const search = false || req.query.search;
+
+  let mainPage = false;
+  let search = false;
+  let searchValue;
+  let backlink = false;
+  if (typeof req.query.search === 'undefined') {
```

Comment:
Maybe we can use `/pwa/search?query=abc`, and differentiate this on the path level instead of parameter.